### ISSUE TITLE
Output FreeSurfer surface area in fsLR space

### ISSUE THREAD

**Issue by tsalo** [nipreps/smriprep#340]
**Is your feature request related to a problem? Please describe.**
This isn't necessarily a problem, but some users, myself included, might like to analyze surface area files coming from Freesurfer, in standard space.

**Describe the solution you'd like**
Now that sMRIPrep warps certain surface outputs (sulcal depth, curvature, and thickness) into fsLR space, I was wondering if surface area (`?h.area.pial` Freesurfer files) could be added to this list.

**Describe alternatives you've considered**
If the surface area file is outputted to the sMRIPrep derivatives in fsnative space as a gifti, we have workflows that warp files like this (e.g., the smoothwm and pial surfaces) to fsLR.

**Additional context**
None

**Comment by effigies** [comment]
I don't think you need anything from us: `mris_convert -c $SUBJECTS_DIR/bert/surf/lh.area.pial $SUBJECTS_DIR/bert/surf/lh.white sub-bert_hemi-L_pialArea.shape.gii`

We do obviously have a workflow for doing this with `curv`/`sulc`/`thickness`, so there's nothing really stopping us from adding it in. I think the better long-term solution is to think of it as an `fmripost` operation, where we write a tool that makes it easy to generate any of these you might want.
